src/Functions/FunctionsAES.h

```diff
@@ -439,7 +439,7 @@ class FunctionDecrypt : public IFunction
         validateFunctionArgumentTypes(*this, arguments,
             FunctionArgumentDescriptors{
                 {"mode", &isStringOrFixedString<IDataType>, isColumnConst, "decryption mode string"},
-                {"input", nullptr, nullptr, "ciphertext"},
+                {"input", &isStringOrFixedString<IDataType>, nullptr, "ciphertext"},
```

Comment:
Why the validator for the third argument remains "nullptr"?

Reply:
Because we don't impose any limitations on column type, it can be either const, nullable or regular.